src/TheDrownedCity/TheWesternWallHelper.ttslua

```diff
@@ -178,6 +178,7 @@ function setup(direction, locationPositions)
           broadcastToAll("Added 'The Inescapable' Encounter Set to the game.", "White")
         else
           trash.putObject(inescapableSet)
+          MythosAreaApi.getEncounterDeckObjects().draw.shuffle() -- this setup has no encounter sets to shuffle in, so shuffle the deck here just in case
```

Comment:
```suggestion
            -- this setup has no encounter sets to shuffle in, so shuffle the deck here just in case
            MythosAreaApi.reshuffleEncounterDeck(true)
```